.github/workflows/test-build.yml

```diff
@@ -130,9 +129,6 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Set up Docker Buildx
```

Comment:
Why remove this action?

Reply:
you right, it shouldn't be removed. i'll undo.